### DISCUSSION TITLE
[vote] Which frameworks you want to support ?

### DISCUSSION THREAD

**Discussion by eeliu** [pinpoint-apm/pinpoint-c-agent#386] [Ideas]
## pre-requirement

- [x]  php 7+
- [x] classloader
- [x] namespace
- [x] *nix

**Comment by eeliu** [comment]
### ThinkPHP5

Documment : https://www.kancloud.cn/manual/thinkphp5/118006

github : https://github.com/top-think/framework/tree/v5.0.24

> In progress

https://github.com/pinpoint-apm/pinpoint-c-agent/tree/dev/plugins/PHP/Framework/ThinkPHP5

**Reply by eeliu** [reply]
- [x] PDO 
- [x] PRedis

**Comment by eeliu** [comment]
## Yii2

Document: https://www.yiiframework.com/
github: https://github.com/yiisoft/yii2

> In progress

https://github.com/pinpoint-apm/pinpoint-c-agent/tree/dev/plugins/PHP/Framework/Yii2

**Comment by eeliu** [comment]
## laravel

github:  https://github.com/laravel/laravel
document： https://laravel.com/

**Reply by eeliu** [reply]
> Here https://github.com/pinpoint-apm/pinpoint-c-agent/tree/dev/plugins/PHP/Framework/laravel

- [x] PDO

**Comment by EyelynSu** [comment]
### Symfony
github: https://github.com/symfony/symfony
doc: https://symfony.com/doc/current/setup.html
